src/azure/cli/_help.py

```diff
@@ -340,6 +352,9 @@ def __init__(self, _data):
         self.text = _data['text']
 
 
+def _get_hanging_indent(max_length, indent):
+    return max_length + (indent * 4) + len(FIRST_LINE_PREFIX)
+
```

Comment:
nit: this could be moved next to get column indent since they are closely related